spec/features/post_show_spec.rb

```diff
@@ -1,19 +1,19 @@
 require 'rails_helper'
 
-RSpec.describe 'Posts', type: :system do
+RSpec.describe 'Posts', type: :feature do
   before :all do
     Like.delete_all
     Comment.delete_all
     Post.delete_all
     User.delete_all
 
-    @fu = User.create(name: 'User one', photo: 'link', bio: 'User one Bio.')
-    @su = User.create(name: 'User two', photo: 'link', bio: 'User two Bio.')
+    @fu = User.created!(name: 'User one', photo: 'link', bio: 'User one Bio.')
+    @su = User.created!(name: 'User two', photo: 'link', bio: 'User two Bio.')
 
-    3.times { |i| Post.create(author: @fu, title: "Post #{i + 1}", text: "First author Post #{i + 1} body") }
-    5.times { |i| Post.create(author: @su, title: "Post #{i + 1}", text: "Second author Post #{i + 1} body") }
+    3.times { |i| Post.created!(author: @fu, title: "Post #{i + 1}", text: "First author Post #{i + 1} body") }
+    5.times { |i| Post.created!(author: @su, title: "Post #{i + 1}", text: "Second author Post #{i + 1} body") }
 
-    6.times { |i| Comment.create(post: @fu.posts.first, user: @fu, text: "Comment #{i + 1}") }
+    6.times { |i| Comment.created!(post: @fu.posts.first, user: @fu, text: "Comment #{i + 1}") }
     3.times { |i| Comment.create(post: @su.posts.first, user: @su, text: "Comment #{i + 1}") }
 
     6.times { Like.create(post: @fu.posts.first, user: @fu) }
```

Comment:
- [ ] [REQUIRED] Kindly add a test that asserts you can see the username of each commentor.
- [ ] [REQUIRED] Kindly add tests that asserts you can see the comment each commentor left.
This hard requirements,let's fix this team.